android/src/main/java/com/squareup/sdk/reader/react/internal/converter/CardConverter.java

```diff
@@ -21,7 +21,7 @@
 import java.util.HashMap;
 import java.util.Map;
 
-class CardConverter {
+public class CardConverter {
```

Comment:
Public so we can reach it from com.squareup.sdk.reader.react, but not exposed API, right?

Reply:
right.